pkg/mcs/resourcemanager/server/token_buckets_test.go

```diff
@@ -144,45 +144,46 @@ func TestGroupTokenBucketRequestLoop(t *testing.T) {
 	gtb.Tokens = 50000
 
 	const timeIncrement = 5 * time.Second
-	const defaultTrickleDuration = int64(time.Second) * 5 / int64(time.Millisecond)
+	const targetPeriod = 5 * time.Second
+	const defaultTrickleMs = int64(targetPeriod) / int64(time.Millisecond)
 
 	// Define the test cases in a table
 	testCases := []struct {
-		requestTokens             float64
-		assignedTokens            float64
-		currentGlobalBucketTokens float64
-		expectedTrickle           int64
+		requestTokens                 float64
+		assignedTokens                float64
+		globalBucketTokensAfterAssign float64
+		expectedTrickleMs             int64
 	}{
-		/* requestTokens, assignedTokens, currentGlobalBucketTokens, trickleTime */
+		/* requestTokens, assignedTokens, globalBucketTokensAfterAssign, trickleTime */
 		{50000, 50000, 0, 0},
-		{50000, 30000, -20000, defaultTrickleDuration},
-		{30000, 15000, -25000, defaultTrickleDuration},
-		{15000, 12500, -27500, defaultTrickleDuration},
-		{12500, 11250, -28750, defaultTrickleDuration},
-		{11250, 10625, -29375, defaultTrickleDuration},
-		// PR_PER_SEC is close to 2000, RU_PER_SEC =  assignedTokens / TrickleTime.
-		{10625, 10312.5, -29687.5, defaultTrickleDuration},
-		{10312.5, 10156.25, -29843.75, defaultTrickleDuration},
-		{10156.25, 10078.125, -29921.875, defaultTrickleDuration},
-		{10078.125, 10039.0625, -29960.9375, defaultTrickleDuration},
-		{10039.0625, 10019.53125, -29980.46875, defaultTrickleDuration},
-		{10019.53125, 10009.765625, -29990.234375, defaultTrickleDuration},
-		{10009.765625, 10004.8828125, -29995.1171875, defaultTrickleDuration},
-		{10004.8828125, 10002.44140625, -29997.55859375, defaultTrickleDuration},
-		{10002.44140625, 10001.220703125, -29998.779296875, defaultTrickleDuration},
-		{10001.220703125, 10000.6103515625, -29999.3896484375, defaultTrickleDuration},
-		{10000.6103515625, 10000.30517578125, -29999.69482421875, defaultTrickleDuration},
-		{10000.30517578125, 10000.152587890625, -29999.847412109375, defaultTrickleDuration},
-		{10000.152587890625, 10000.0762939453125, -29999.9237060546875, defaultTrickleDuration},
-		{10000.0762939453125, 10000.038146972656, -29999.961853027343, defaultTrickleDuration},
+		{50000, 30000, -20000, defaultTrickleMs},
+		{30000, 15000, -25000, defaultTrickleMs},
+		{15000, 12500, -27500, defaultTrickleMs},
+		{12500, 11250, -28750, defaultTrickleMs},
+		{11250, 10625, -29375, defaultTrickleMs},
+		// RU_PER_SEC is close to 2000, RU_PER_SEC =  assignedTokens / TrickleTime.
```

Comment:
Please add the unit conversion

```suggestion
		// RU_PER_SEC is close to 2000, RU_PER_SEC =  assignedTokens / (TrickleTime / 1000).
```